packages/nx/src/command-line/connect/connect-to-nx-cloud.ts

```diff
@@ -51,12 +52,39 @@ export async function connectToNxCloudIfExplicitlyAsked(
 
 export async function connectToNxCloudCommand(): Promise<boolean> {
   const nxJson = readNxJson();
+
+  const githubSlug = getGithubSlugOrNull();
+
   if (isNxCloudUsed(nxJson)) {
+    const apiUrl =
+      process.env.NX_CLOUD_API ||
+      process.env.NRWL_API ||
+      `https://cloud.nx.app`;
+    let connectCloudUrl;
+    if (githubSlug) {
+      connectCloudUrl = `${apiUrl}/setup/connect-workspace/vcs?provider=GITHUB&selectedRepositoryName=${encodeURIComponent(
+        githubSlug
+      )}`;
+    } else if (
+      process.env.NX_CLOUD_ACCESS_TOKEN ||
+      !!nxJson.nxCloudAccessToken
+    ) {
+      const token =
+        process.env.NX_CLOUD_ACCESS_TOKEN || nxJson.nxCloudAccessToken;
+      connectCloudUrl = `https://cloud.nx.app/setup/connect-workspace/manual?accessToken=${token}`;
+    } else {
+      throw new Error(
+        `Unable to authenticate. Either define accessToken in nx.json or set the NX_CLOUD_ACCESS_TOKEN env variable.`
+      );
+    }
+
     output.log({
       title: '✔ This workspace already has Nx Cloud set up',
       bodyLines: [
         'If you have not done so already, connect your workspace to your Nx Cloud account:',
-        `- Login at ${getNxCloudUrl(nxJson)} to connect your repository`,
+        `- Do so by going to the following URL: 
```

Comment:
I'd reword this to be more terse.

`- Connect with Nx Cloud at ${connectCloudUrl}`

Just to reinforce what going to the URL will do.